inc/vcf/fixer.hpp

```diff
@@ -174,29 +232,108 @@ namespace ebi
         }
 
       protected:
+
+        size_t split_and_find(std::string line, std::string separator, std::string value) {
```

Comment:
in next commit I will use const references and there will be a documentation comment.

Some context on usage: if we have `"GT:AC:AN    0/0:3:4"` and have to remove the AC field, we need to know that it's in position 1 for later when we split the sample column: `["0/0", "3", "4"]`. 

`split_and_find(format_column, ...)` returns that position 1 from the FORMAT column, or `format_column.npos` if it was not found (just as in `string::find` methods).
We cannot return iterators because those will be pointing to the split `vector<string>` local to `split_and_find`.